MapboxVisionTests/VisionManagerTests.swift

```diff
@@ -61,4 +61,62 @@ class VisionManagerTests: XCTestCase {
             "VisionManager should be successfully deallocated after calling destroy and releasing the object"
         )
     }
+
+    func testPerformanceConfigMergedTranslatesToSamePerformance() {
+        // Given
+        // object of VisionManager
+
+        // When
+        // model performance config is set
+        let performance = ModelPerformance(mode: .dynamic, rate: .high)
```

Comment:
I would suggest to move this line into `Given` section.
We test the behavior for `modelPerformanceConfig`.
Does that make sense?

Reply:
Good point, applied.